edalize/templates/symbiflow/symbiflow-vpr-makefile.j2

```diff
@@ -1,14 +1,25 @@
 #Auto generated by Edalize
+SHELL = bash
 
 TOP	:= {{ top }}
 VERILOG := {{ sources }}
 PARTNAME:= {{ partname }}
-DEVICE  := {{ part }}_test
+DEVICE  := {{ part }}_{{ device_suffix }}
 BITSTREAM_DEVICE := {{ bitstream_device }}
 SDC	:= {{ sdc }}
 PCF	:= {{ pcf }}
 XDC	:= {{ xdc }}
 BUILDDIR:= {{ builddir }}
+TOOLCHAIN_PREFIX := {{ toolchain_prefix }}
+{% if toolchain_prefix -%}
```

Comment:
IMO It would probably be better to have the vendor to drive the `if` instead of toolchain_prefix